Wireframe.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <Title>Purpose of a Wireframe</Title>
+</head>
+<link rel="stylesheet" type="text/css" href="CSS1.css" />
+
+<body>
+    <header>
+        Wireframe

```

Comment:
This text, in my opinion, is the main title of this page. Which html tag would you use for the main title?